### ISSUE TITLE
Add exception bubble up example in Exception Catching chapter

### ISSUE THREAD

**Issue by nununo** [SAP/styleguides#234]
In the [Exceptions catching chapter](https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#catching) I believe it would be useful to provide an example of [exception bubbling], making it clear that, while exception throwing is to be done any levels, handling exceptions should only be done at the highest level possible, making it clear that exceptions which are left unhandled at a given level will automatically bubble up to the next (depending on them being declared and from which top exception they inherit, of course).

**Comment by pokrakam** [comment]
Is this not adequately described [here](https://github.com/SAP/styleguides/blob/main/clean-abap/sub-sections/Exceptions.md)?

**Comment by nununo** [comment]
Hum... indeed it is. But now I'm confused. I was fully convinced that the [main document](https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md) contained all the information and never really explored the sub-sections.

I can't find the [text you linked](https://github.com/SAP/styleguides/blob/main/clean-abap/sub-sections/Exceptions.md) in the main document. Will the sub-sections be incorporated in the main document at some point in the future?

**Comment by pokrakam** [comment]
It's easy to miss, the only link is at the end of
https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#throw-cx_static_check-for-manageable-exceptions
It's worth exploring the sub-sections directly from here
https://github.com/SAP/styleguides/tree/main/clean-abap/sub-sections

The guide could probably benefit from a bit of restructuring, it seem like we have 90% in one huge section and 10% in a handful of subsections.

**Comment by nununo** [comment]
> The guide could probably benefit from a bit of restructuring, it seem like we have 90% in one huge section and 10% in a handful of subsections.

I fully agree. I rely heavily on Clean ABAP, have read through it several times, and only now (thanks to you) did I find that the monolithic text isn't the whole of it!

And that link is really easy to miss because, since there is no distinction between internal and external links, I was convinced it was an external link to another page by Uncle Bob.
